src/test/javascript/spec/service/faq.service.spec.ts

```
@@ -174,6 +174,27 @@ describe('Faq Service', () => {
+            const category = {
+                color: '#6ae8ac',
+                category: 'category1',
+            } as FaqCategory;
+            const returnedFromService = { categories: [JSON.stringify(category)] };
+            const expected = { ...returnedFromService };
+            const courseId = 1;
+            const faqState = FaqState.ACCEPTED;
+            service
+                .findAllCategoriesByCourseIdAndCategory(courseId, faqState)
+                .pipe(take(1))
+                .subscribe((resp) => (expectedResult = resp));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)